src/utils/error.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Extract an error from an Axios error response.
+ *
+ * @param err - The error from Axios.
+ * @returns - The extracted response error.
+ */
+export function extractResponseError(
+  // eslint-disable-next-line
+  err: any
+): Error {
+  /* istanbul ignore next */
+  if (!err) {
+    return new Error("Invalid response error");
```

Comment:
This error message is a little confusing. I would interpret it as "unknown/unexpected error" and not as "we expected to see an error here but there is none".